roles/pulp_webserver/templates/nginx.conf.j2

```diff
@@ -76,7 +76,7 @@ http {
 
         # The default client_max_body_size is 1m. Clients uploading
         # files larger than this will need to chunk said files.
-        client_max_body_size {{ pulp_client_max_body_size }};
```

Comment:
@amittar-lb please remove this commit and set the following env var to ansible playbook

pulp_client_max_body_size = "3G"

Reply:
Done